src/main/java/use_cases/game_preparation_use_case/GamePrepOutputBoundary.java

```diff
@@ -0,0 +1,42 @@
+package use_cases.game_preparation_use_case;
+
+import java.util.List;
+
+/**
+ * GamePrepOutputBoundary provides the interface for the presenter to control
+ * the UI of the game.
+ */
+public interface GamePrepOutputBoundary {
+
+    /**
+     * Display a screen to ask both players for their names and let them choose
+     * the creatures for their game.
+     *
+     * @param creatureNames the names of the creatures that can be selected for
+     *                      the game.
+     */
+    void showGamePreparationScreen(List<String> creatureNames);
+
+    /**
+     * Notify the players that the game creation process cannot go through due
+     * to unexpected inputs.
+     * This could be empty player names, or not exactly 3 selected creatures.
```

Comment:
I believe more cases for unexpected inputs should be considered for usernames. Refer to comment one above this